src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelectorAll('li');
+
+function sortList(list) {
+  function strToNum(str) {
+    return +str.slice(1).split(',').join('');
+  }
```

Comment:
Move this function outside of sortList function, to avoid creating this function on each call of sort list.